financialaid/models.py

```diff
@@ -138,7 +106,11 @@ class FinancialAid(TimestampedModel):
     tier_program = models.ForeignKey(TierProgram, null=False)
     status = models.CharField(
         null=False,
-        choices=[(status, status) for status in FinancialAidStatus.ALL_STATUSES],
+        choices=[
+            (status, status) for status in FinancialAidStatus.ALL_STATUSES
+            # The rejected status has been temporarily disabled as a valid status for FinancialAid
```

Comment:
It's not a "temporary" change per se - it's supposed to be removed for 10/15, but I remember during a standup last week @pdpinch said to leave the code in for `rejected`, but commented out.

But you're right, we should remove `rejected` with a separate PR - will revert changes and do that there.